protocol/0028-GOVE-governance.md

```diff
@@ -680,15 +689,29 @@ It is NOT possible to submit a governance proposal where the source account is t
 
 ##### Batch Proposals
 
-- A batch proposal containing one or more component submissions for each type of proposal term can be submitted and is accepted as a valid proposal. (<a name="0028-GOVE-146" href="#0028-GOVE-146">0028-GOVE-146</a>)
-
-- A batch proposal containing component submissions with different categories will be rejected with an informative error message. (<a name="0028-GOVE-147" href="#0028-GOVE-147">0028-GOVE-147</a>)
+- A batch proposal containing one or more component submissions mixing different proposal types can be submitted and is accepted as a valid proposal. (<a name="0028-GOVE-146" href="#0028-GOVE-146">0028-GOVE-146</a>)
 
-- A batch proposal submitted with component submissions having the same category but different closing timestamps will be rejected with an informative error message. (<a name="0028-GOVE-148" href="#0028-GOVE-148">0028-GOVE-148</a>)
+- A batch proposal submitted with component submissions having different closing timestamps will be rejected with an informative error message. (<a name="0028-GOVE-148" href="#0028-GOVE-148">0028-GOVE-148</a>)
 
-- A batch proposal submitted with component submissions having the same category and the same closing timestamps but different enactment timestamps will be accepted and move to voting.  (<a name="0028-GOVE-149" href="#0028-GOVE-149">0028-GOVE-149</a>)
+- A batch proposal submitted with component submissions with different enactment timestamps will be accepted and move to voting.  (<a name="0028-GOVE-149" href="#0028-GOVE-149">0028-GOVE-149</a>)
    1. If this proposal is accepted, each of the components will be enacted at the time of their differing enactment timestamps. (<a name="0028-GOVE-145" href="#0028-GOVE-145">0028-GOVE-145</a>)
 
+- A batch proposal containing 
+1. freeform proposal,
+1. an asset proposal, 
```

Comment:
Could we remove assets proposals from the list here, they require the extra validation period which is meant to check that the code at the address on ethereum is actually a ERC20 token.

Reply:
@davidsiska-vega to clarify that we will include asset proposals, however, due to the pre-validation it may cause delays to the full batch.

Reply:
I thought we agreed to include only update asset proposals not new asset proposals?

Reply:
Yes, this was a comment decided before a further decision was made. 

To clarify the spec is to include only update asset proposals.

Reply:
Added / resolved.